compiler/ir/ir.psi2ir/build.gradle.kts

```diff
@@ -8,6 +8,8 @@ dependencies {
     compile(project(":compiler:frontend"))
     compile(project(":compiler:frontend.java"))
     compile(project(":compiler:backend-common"))
+    compile(project(":compiler:ir.backend.common"))
+    compile(project(":compiler:backend"))
```

Comment:
Psi2ir is the common part of all backends, so it shouldn't depend on the old JVM backend. Please refactor to get rid of this dependency. Note that since the code fragment codegen logic is needed only on JVM for now, you can put it by default into `backend.jvm` (maybe with some gluing parts in `backend.jvm.entrypoint`)

Reply:
Thanks for taking a look, Alex! It should be doable to get rid of the dependency, it's mostly data types and structures being shared, so can probably just lift those higher in the build graph.

Will get to it!